src/Cli/AzureCliLogin.ts

```diff
@@ -30,6 +31,7 @@ export class AzureCliLogin {
 
         await this.executeAzCliCommand(["version"], true, execOptions);
         core.debug(`Azure CLI version used:\n${output}`);
+        this.azVersion = JSON.parse(output)["azure-cli"];

```

Comment:
Have you checked all Azure CLI version to see the output format of this? What if `JSON.parse` throws an exception?

Reply:
Azure CLI hasn't changed its JSON output format for over 5 years.
![image](https://github.com/user-attachments/assets/bb542af5-5453-4fb0-b455-06918e266f4b)
I prefer to use `az version` to get the Azure CLI version. If `JSON.parse` throws an exception, it means the user is using an unsupported version of Azure CLI and should be rejected.

@jiasli, do you have any other suggestions for reliably obtaining the Azure CLI version?

Reply:
`az version` is stable. It is safe to use it.